assets/cubyz/biomes/prairie/dry_spell.zig.zon

```diff
@@ -0,0 +1,66 @@
+.{
+	.properties = .{
+		.dry,
+	},
+	.minRadius = 16,
+	.maxRadius = 32,
+	.chance = 0,
+	.minHeight = 35,
+	.maxHeight = 40,
+	.roughness = 1,
+	.hills = 3,
+	.ground_structure = .{
+		"4 to 5 cubyz:limestone",
+	},
+	.parentBiomes = .{
+		.{
+			.id = "cubyz:prairie/base",
+			.chance = 24,
```

Comment:
Could you lower the chance a bit? It does seem to make some of the prairie biomes rather dry.